### PR TITLE
[IMP] general: add new doc about payment acquirers

### PR DESCRIPTION
**Pull request by jcs-odoo**
- Updates the main current doc about Payment Acquirers with a completely new doc
- Moves some docs from ecommerce to general
- redirects

  - ecommerce/shopper_experience/payment --> general/payment_acquirers/payment_acquirers
  - ecommerce/shopper_experience/authorize --> general/payment_acquirers/authorize
  - ecommerce/shopper_experience/paypal --> general/payment_acquirers/paypal
  - ecommerce/shopper_experience/wire_transfer --> general/payment_acquirers/wire_transfer

### CODE REVIEW

File: general/payment_acquirers/payment_acquirers.rst [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+  - | :doc:`Wire Transfer <wire_transfer>`
+    | When selected, Odoo displays your payment information with a payment reference. You have to
+      approve the payment manually once you have received it on your bank account.
+  - | SEPA Direct Debit
+    | Your customers can sign a SEPA Direct Debit mandate online and get their bank account charged
+      Directly. :doc:`Click here <../../accounting/receivables/customer_payments/batch_sdd>` for
+      more information about this payment method.
+
+- Online Payment Providers
```
Comment (Lau-Odoo) [inline general/payment_acquirers/payment_acquirers.rst:42]:
```suggestion
- Online Payment Providers:
```

File: general/payment_acquirers/payment_acquirers.rst [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  - Ingenico
+  - :doc:`Paypal <paypal>`
+  - PayU
+  - PayUmoney
+  - Sips
+  - Stripe
+
+.. note::
+   Some of these Online Payment Providers can also be added as :doc:`Bank Accounts
+   <../../accounting/bank/setup/bank_accounts>`, but this is *not* the same process as adding them
```
Comment (Lau-Odoo) [inline general/payment_acquirers/payment_acquirers.rst:57]:
```suggestion
   <../../accounting/bank/setup/bank_accounts>`, but this is **not** the same process as adding them
```

**Comment by jcs-odoo** [comment]
@Lau-Odoo I accepted only the second suggestion. 
Can you approve this PR if it's good for you? Thanks :)

**Comment by jcs-odoo** [comment]
@jubodoo 
Ready for final review it seems.

File: general/payment_acquirers/payment_acquirers.rst [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+Save and reuse Credit Cards
+***************************
+
+With the **Save Cards** feature, Odoo can store **Payment Tokens** in your database, which can be
+used for subsequent payments, without having to reenter the payment details. This is particularly
+useful for subscriptions' recurring payments.
+
+.. note::
+   The payment tokens stored on the database don't hold information about the credit cards used.
```
Comment (dbo-odoo) [inline general/payment_acquirers/payment_acquirers.rst:109]:
```suggestion
   No credit card number gets directly saved in database; instead, payment tokens store a unique reference to the card which is saved on the payment acquirer's servers. This reference is useless without your credentials for the acquirer.
```
we do hold information (e.g. last 4 digits)

File: general/payment_acquirers/payment_acquirers.rst [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+Place a hold on a card
+**********************
+
+If the **Capture Amount Manually** field is enabled, the funds are reserved for a few days on the
```
Comment (dbo-odoo) [inline general/payment_acquirers/payment_acquirers.rst:114]: there should be a way to show that this is not available for all providers; ideally, a table for supported features somewhere per acquirer:
|               | Capture Manually | Save Cards | Pay from Odoo | Redirect to acquirer website |
|---------------|------------------|------------|---------------|------------------------------|
|  Adyen        |                  |            |               | x                            |
| Alipay        |                  |            |               | x                            |
| Authorize.net | x                | x          | x             | x                            |
| Buckaroo      |                  |            |               | x                            |
| Ingenico      |                  | x          | x             | x                            |
| Paypal        |                  |            |               | x                            |
| PayUMoney     |                  |            |               | x                            |
| SIPS          |                  |            |               | x                            |
| Stripe        |                  | x          | x             | x                            |

Comment (jcs-odoo) [reply general/payment_acquirers/payment_acquirers.rst:114]: Thanks for the table @dbo-odoo !!! Huge thanks :)

Comment (dbo-odoo) [reply general/payment_acquirers/payment_acquirers.rst:114]: wait i messed it up :

Comment (dbo-odoo) [reply general/payment_acquirers/payment_acquirers.rst:114]: there, correct now

Comment (jcs-odoo) [reply general/payment_acquirers/payment_acquirers.rst:114]: alright :)
Thanks

File: general/payment_acquirers/payment_acquirers.rst (hunk 3 above)
Comment (jcs-odoo) [reply general/payment_acquirers/payment_acquirers.rst:109]: I modified the note in the intro of the doc and removed it from this section. It was redundant.

> Odoo does not keep Credit Card numbers or credentials on its servers, nor is it stored on Odoo databases hosted elsewhere. Instead, Odoo apps use a unique reference to the data stored in the payment acquirers’ systems, where the information is safely stored. This reference is useless without your credentials for the payment acquirer.

**Pull request merged**